install.sh

```diff
@@ -81,17 +81,32 @@ sudo ln -s `pwd`/crew $CREW_PREFIX/bin
 mkdir $CREW_LIB_PATH/lib && cd $CREW_LIB_PATH/lib
 wget -N -c $URL/lib/package.rb
 wget -N -c $URL/lib/package_helpers.rb
-#create the device.json file
+
+#create or update the device.json file
+ruby_version='2.0.0p247'
```

Comment:
OK, so let's not update device.json then.  It makes no sense to store a version that does not match the same package.  Consistency matters.  If we hardcode versions all over the place, it makes the whole process much more difficult to maintain.  I see nothing wrong with testing if the entire installation process with the version inside ruby.rb.  What kind of instabilities are introduced any time you upgrade any package?